### ISSUE TITLE
Unexpected lemma for "coding" -> "cod"

### ISSUE THREAD

**Issue by aleksandertym** [explosion/spaCy#2257]
I think it should be "code" instead of "cod". It happens in more cases e.g. "riding" -> "rid" (should be "ride").

```
import spacy

nlp = spacy.load("en")

print([x.lemma_ for x in nlp("I like coding")])
```
output:

`['-PRON-', 'like', 'cod']`

* **spaCy version:** 2.0.11
* **Models:** en
* **Platform:** Linux-4.13.0-38-generic-x86_64-with-Ubuntu-16.04-xenial
* **Python version:** 3.5.2

**Comment by bellabie** [comment]
I had this same issue for coded --> cod.
Changing _verbs_irreg.py seemed to fix:
`"coded": ("code",),
"coding": ("code",),`

Should I list as a separate issue?

**Comment by ines** [comment]
Fixed on the `develop` branch (see 2ec2192000e99414a7bbbf11c069c92606da4dc5) and will be included in the upcoming `v2.1.x`.
